### ISSUE TITLE
openmediavault-upgrade-4.7 - pre.d

### ISSUE THREAD

**Issue by SaSha-ShaS** [dleidert/openmediavault-upgrade#41]
Hi,

I have an issue with openmediavault-upgrade-4.7. From the log file below, I can see that I might have errors in pre.d:

- run-parts: executing /root/openmediavault-upgrade-4.7/pre.d/15-stop-services
Failed to get unit file state for docker.service: No such file or directory

- run-parts: executing /root/openmediavault-upgrade-4.7/pre.d/20-remount-ro
- run-parts: /root/openmediavault-upgrade-4.7/pre.d/20-remount-ro exited with return code 11

[omv_release_upgrade_to_buster.2021-08-09-160516.log](https://github.com/dleidert/openmediavault-upgrade/files/6955986/omv_release_upgrade_to_buster.2021-08-09-160516.log)

Can you help?

**Comment by learning-more** [comment]
From your log: 
> Re-mounting /srv/dev-disk-by-id-ata-ST31000520AS_5VX086CN-part3 read-only
> Remounting is not supported at present. You have to umount volume and then mount it once again.

I have this same issue also. From my understanding, you need to unmount the disks, reboot and then try again. Disks are unable to be unmounted if they are being actively used by applications...so it is important to follow the error instructions "quit samba, etc."

Also, there was a news posting on the OMV page about OMV 5 fixing an issue with jmicro-related USB enclosures... but this isn't my issue- my drives are internal.

I'll report back if I've found a solution.

**Comment by SaSha-ShaS** [comment]
I already disabled all services via the OMV control panel, except SSH. Unmounting the named device seemed to be the solution for performing the upgrade. As it was not possible to perform this via the OMV control panel, I used the following command from the terminal:

`umount /srv/dev-disk-by-id-ata-ST31000520AS_5VX086CN-part3`

I saw no response and was not able to check whether this succeeded or not. But ultimately, upgrade to version 5.6.14 succeeded. Whether everything works as before I will check now. What I already recognized is that the network folders are again reachable, whereas MySQL is not available anymore. I will install it again and retry.

Thank you for help.
